orion/pkg/repos/migrations/000003_create_table_achievements.up.sql

```diff
@@ -1,11 +1,12 @@
 CREATE TABLE achievements
 (
-    id         int unsigned NOT NULL AUTO_INCREMENT,
-    created_at datetime     NOT NULL,
-    updated_at datetime     NOT NULL,
-    deleted_at datetime,
-    year       int unsigned NOT NULL,
-    message    text         NOT NULL,
+    id           int unsigned NOT NULL AUTO_INCREMENT,
+    created_at   datetime     NOT NULL,
+    updated_at   datetime     NOT NULL,
+    deleted_at   datetime,
+    published_at datetime,
```

Comment:
Just as practice for people, now that everybody has a good amount of data on their local environments (i.e. everybody is on migration 7-8 or whatever), if you alter migration version 3 that means everybody would have to start over from version 3 and build up again which means blowing their databases.

Instead, leave this migration intact and create a new migration file that only adds a column to this achievements table; Similarly the `.down.sql` file should remove this column